app/controllers/sign_up/registrations_controller.rb

```diff
@@ -18,6 +18,7 @@ def new
       @register_user_email_form = RegisterUserEmailForm.new
       session[:sign_up_init] = true
       analytics.track_event(Analytics::USER_REGISTRATION_ENTER_EMAIL_VISIT)
+      render :new, locals: { request_id: nil }
```

Comment:
anything in the view is going to inherently have `to_s` called on it. So this will render in the hidden input field as `''`, I believe